admin-dev/themes/new-theme/scss/components/_modal.scss

```
@@ -0,0 +1,62 @@
+/*!
+ * 2007-2020 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *
+ */
+.modal-iframe {
+  .modal-dialog {
+    width: 90%;
+    max-width: 1200px;
+    max-height: 95%;
+    margin: 0 auto;
+
+    .modal-content {
+      height: 100%;
+      padding: 0;
+      margin: 0;
+
+      .modal-body {
+        padding: 0;
+
+        .modal-iframe-loader {
+          position: absolute;
+          z-index: 1;
+          display: flex;
+          align-items: center;
+          justify-content: center;
+          width: 100%;
+          height: 100%;
+          min-height: 80px;
+          background: rgba(255, 255, 255, 1);
+        }
+
+        iframe {
+          width: 100%;
+          height: 100%;
+          border: none;
+          scroll-behavior: auto;
+        }
+      }
+    }
+  }
+}
```

```suggestion
$component-name: modal;

.#{$component-name}-iframe {
  .${$component-name}-dialog {
    width: 90%;
    max-width: 1200px;
    max-height: 95%;
    margin: 0 auto;

    .${$component-name}-content {
      height: 100%;
      padding: 0;
      margin: 0;

      .${$component-name}-body {
        padding: 0;

        iframe {
          width: 100%;
          height: 100%;
          border: none;
          scroll-behavior: auto;
        }
      }
    }
  }

  &-loader {
    position: absolute;
    z-index: 1;
    display: flex;
    align-items: center;
    justify-content: center;
    width: 100%;
    height: 100%;
    min-height: 80px;
    background: rgba(255, 255, 255, 1);
  }
}
```

If you find it interesting, let me know, overwise, resolve the comment

---

I tried again, I remember now why it didn't work The scss linter forces me to set `!default` however when I do that it doesn't work anymore I wonder if it is because the variable is already defined in several other scss files?

Anyway, I just added a comment to ignore this rule here and it works ok now!